spatial_maps/fields.py

```diff
@@ -11,6 +11,8 @@ def find_peaks(image):
     -------
     peaks : array
         coordinates for peaks in image as [row, column]
+    :Authors:
+        Halvard Sutterud <halvard.sutterud@gmail.com>
```

Comment:
Might be my bad memory, but I am pretty sure I wrote this function as part of the field variability notebook :smile:

Reply:
that is probably correct, it looks reminiscent of something I've written so I assumed it was mine :)